src/InlineEntityFormControllerInterface.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\inline_entity_form\InlineEntityFormControllerInterface.
+ */
+
+namespace Drupal\inline_entity_form;
+
+use Drupal\Component\Plugin\ConfigurablePluginInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Defines the interface for entity browser widgets.
+ */
+interface InlineEntityFormControllerInterface extends ConfigurablePluginInterface {
+
+  /**
+   * Returns an array of css filepaths for the current entity type, keyed
+   * by theme name.
+   *
+   * If provided, the "base" CSS file is included for all themes.
+   * If a CSS file matching the current theme exists, it will also be included.
+   *
+   * @code
+   * return [
+   *   'base' => drupal_get_path('module', 'test_module') . '/css/inline_entity_form.base.css',
+   *   'seven' => drupal_get_path('module', 'test_module') . '/css/inline_entity_form.seven.css',
+   * ];
+   * @endcode
+   *
+   * @return array
+   *   List of css files for inclusion keyed by theme name.
+   */
+  public function css();
```

Comment:
I guess we need to replace this with libraries? Because its not possible to add individual css files during rendering.